cmd2/cmd2.py

```diff
@@ -2866,11 +2871,17 @@ def default(self, statement: Statement) -> Optional[bool]:  # type: ignore[overr
             return self.do_shell(statement.command_and_args)
         else:
             err_msg = self.default_error.format(statement.command)
-
+            if self.suggest_similar_command:
+                suggested_command = self._suggest_similar_command(statement.command)
+                if suggested_command:
+                    err_msg = err_msg + os.linesep + self.default_suggestion_message.format(suggested_command)
```

Comment:
With my alternate suggestion message, no need for line separator.

Reply:
I can use a space